shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.shadow.route.engine.util;
+
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.expr.simple.LiteralExpressionSegment;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class ShadowExtractorTest {
+
+    @Test
+    public void extractValuesIsSimpleExpressionSegment() {
+        LiteralExpressionSegment LiteralExpressionSegment = new LiteralExpressionSegment(1, 2, "");
+        List<Object> parametersList = new LinkedList<>();
+        Collection<Comparable<?>> result = new LinkedList<>();
+        result.add((Comparable<?>) LiteralExpressionSegment.getLiterals());
+        assertThat(ShadowExtractor.extractValues(LiteralExpressionSegment, parametersList), is(Optional.of(result)));
```

Comment:
AssertThat, please use the exact value，example:

```java
assertTrue(ShadowExtractor.extractValues(simpleExpressionSegment, parametersList).isPresent());
assertThat(ShadowExtractor.extractValues(simpleExpressionSegment, parametersList).get().iterator().next(), is("expected"));
```

Reply:
Ok,thank you very much.